cf/rds-agent.yaml

```diff
@@ -0,0 +1,98 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up EC2 instance, can be accessed through SSM for connecting to RDS. For more information read https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager.html
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
```

Comment:
development? We don't use this environment in iac

Reply:
I copied this over from redis, which also has this and as default too, should I change redis too?

Reply:
`development` is needed by inframock. in the case of redis it isn't used because we don't run redis locally. I am unsure if we need the `rds-agent` locally, probably not if `postgre` is running in a container.

Reply:
I think we do run redis locally, if you check in inframock a container is being set up

Reply:
Localstack's redis was part of the premium version, so we start a service on our own.

Reply:
Yes, we do run redis locally, we always did, because we didn't want to pay for premium version of localstack.
It looks like we only use it for s3, dynamo and sns: https://github.com/hms-dbmi-cellenics/inframock/blob/master/src/app.py#L79